datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/__init__.py

```diff
@@ -34,6 +36,7 @@
     recommended_monitors,
     saved_views,
     service_checks,
+    package,
```

Comment:
nit: commands and imports should be in alphabetical order

Reply:
Good call, looks like isort was failing on this too. Just fixed
It originally was in alphabetical order when I had the command named `setup` 😆

Reply:
even smaller nit: package should also be moved up in `ALL_COMMANDS` 😆

Reply:
haha, my bad, just fixed